### ISSUE TITLE
Feature request: Subfolder for Saves

### ISSUE THREAD

**Issue by Defrago** [rr-/Tomb1Main#87]
Place Saves in subfolders. Maybe also move Saves inside the game folder to each subfolder on game startup (for users migrating from TombATI and older T1M releases). TR2Main naming:

![Снимок (4)](https://user-images.githubusercontent.com/25227941/111526671-15fe3400-8770-11eb-9498-6de1941bcab9.PNG)

**Comment by Defrago** [comment]
Also folder(s) for JSON configuration. Probably name it _jsonconfig_, _configuration_ or _cfg_

**Comment by rr-** [comment]
Screenshots are not possible without #57

**Comment by walkawayy** [comment]
A screenshot folder is probably easy to implement now. But is there an easy cross-platform solution for folder separators in C?

**Comment by rr-** [comment]
>But is there an easy cross-platform solution for folder separators in C?

Yes, just go with `/`, Windows should handle that.
(Until the user tries to run the game from C:\ゲーム\ in which case we're screwed anyway.)

**Comment by walkawayy** [comment]
Nice. I made screenshots save to a folder, but moving saves is a bigger deal since right now they are expected in the main folder. Idk what your thoughts are on that. It's also not a pressing issue or anything.

**Comment by rr-** [comment]
I'd say that we should keep the loop but check in both folders: first the saves/ directory, then if there's no save in the saves/ directory, fall back to the main game directory for those who didn't know or neglected to move the saves. And save always to saves/. For now I'd keep saveati.%d naming and consider changing this only as part of #277.

**Comment by walkawayy** [comment]
Good idea. I'll leave saves on hold until we restructure them.

I'll open a PR for the screenshot folder to close issue #255.

**Comment by walkawayy** [comment]
Screenshots are done and now go into a subfolder. Updating the issue to mention only saves which still depends on #277 first.

**Comment by Richard-L** [comment]
Should UB saves go into a folder named as such, so **savesUB**, instead of **savesGold**? The whole "Gold" thing was mainly a marketing concern to re-retail the game from what I understand. Speedrun.com also name their page "trub" https://www.speedrun.com/trub, also this isn't very consistent.
